### PR TITLE
modified:   chapter_deep-learning-computation/model-construction.md

### PR DESCRIPTION
**Pull request by xinetzone**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-629/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-629/

File: chapter_deep-learning-computation/model-construction.md [hunk 1]
```diff
@@ -163,8 +163,12 @@ net(X)
 ```{.python .input}
 class MySequential(nn.Block):
     def add(self, block):
-    # 这里，`block`是`Block`子类的一个实例，我们假设它有一个唯一的名称。我们把它保存在'Block'类的成# 员变量`_children` 中。`block`的类型是OrderedDict。当`MySequential`实例调用`initialize`
-    # 函数时，系统会自动初始化`_children`的所有成员
+        '''
```
Comment (astonzhang) [inline chapter_deep-learning-computation/model-construction.md:166]: For consistency, we do not use doc strings here throughout the book.

**Comment by mli** [comment]
Job d2l-zh/PR-629/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-629/

**Comment by mli** [comment]
Job d2l-zh/PR-629/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-629/

**Comment by mli** [comment]
Job d2l-zh/PR-629/7 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-629/

**Comment by mli** [comment]
Job d2l-zh/PR-629/8 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-629/

**Comment by mli** [comment]
Job d2l-zh/PR-629/9 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-629/

**Pull request closed without merging**